drivingRange/NumToText/index.js

```diff
@@ -0,0 +1,90 @@
+const numberToText = [
+  "one",
+  "two",
+  "three",
+  "four",
+  "five",
+  "six",
+  "seven",
+  "eight",
+  "nine"
+];
+const teenToText = [
+  "ten",
+  "eleven",
+  "twelve",
+  "thirteen",
+  "fourteen",
+  "fifteen",
+  "sixteen",
+  "seventeen",
+  "eighteen",
+  "nineteen"
+];
+const writeNumber = number => {
+  var tempNumber = number;
+  if (number < 10) {
+    console.log(numberToText[number - 1]);
+  }
+  if (number > 9 && number < 20) {
+    console.log(teenToText[number - 10]);
+  }
+  if (number > 19 && number < 30) {
+    if (number == 20) {
```

Comment:
when you start seeing patterns like this throughout, these are perfect candidates to figure out how to write a function or set up a check once, that can be applied for several number cases